api_poll/send_scan_data_to_tube.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import beanstalkc
```

Comment:
where would `beanstalkc` be imported from? There is no beanstsalkc.py in `api_poll` module.
```
from scanning.vendors import beanstalkc
```
should work.